src/passes/RoundTrip.cpp

```diff
@@ -55,14 +55,12 @@ struct RoundTrip : public Pass {
     writer.setDebugInfo(runner->options.debugInfo);
     writer.write(*module, tempName);
     // Read
-    Module newModule;
+    ModuleUtils::clearModule(*module);
     ModuleReader reader;
-    reader.read(tempName, newModule);
+    // TODO: enable debug info when relevant
+    reader.read(tempName, *module);
     // Clean up
     std::remove(tempName.c_str());
-    // Swap in
-    ModuleUtils::clearModule(*module);
-    ModuleUtils::copyModule(newModule, *module);
```

Comment:
Can we get rid of `ModuleUtils::copyModule` entirely? IIRC, it isn't used anywhere else.

Reply:
Hmm, maybe it'll be useful in the future? Seems a shame to remove it (but unfortunate it isn't tested).